src/components/schemas/expense.yaml

```diff
@@ -531,13 +531,122 @@ expense_in:
       $ref: './fields.yaml#/custom_fields'
 
 spender_expense_in:
-  allOf:
-    - type: object
-      required: [ 'claim_amount' ]
-      properties:
-        claim_amount:
-          $ref: './fields.yaml#/amount'
-    - $ref: '#/expense_in'
+  type: object
+  additionalProperties: false
+  properties:
+    id:
+      $ref: './fields.yaml#/id_string'
+    spent_at:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+      description: |
+        signifies when this expense was made.
+        This date cannot be in future.
+    source:
+      type: string
+      description: |
+        The name of the client app which is creating the expense.
+        if its a thrid party app, it prefixed with TPA_.
+        Ex - If you pass SLACK_APP, it will be saved as TPA_SLACK_APP. If not
+        passed, source is saved as TPA.
+      example: SLACK_APP
+    merchant:
+      type: string
+      maxLength: 255
+      nullable: true
+      description: |
+        String denoting the name of the vendor to which the Expense was made.
+      example: Uber
+    foreign_currency:
+      $ref: './fields.yaml#/foreign_currency'
+    foreign_amount:
+      $ref: './fields.yaml#/foreign_amount'
+    purpose:
+      type: string
+      example: Team lunch
+    cost_center_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      nullable: true
+      example: 1049
+    category_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      nullable: true
+      example: 49058
+    project_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      nullable: true
+      example: 92895
+    source_account_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      example: acwbl222wlg
+    tax_amount:
+      type: number
+      nullable: true
+      description: Tax amount as setup by your organization.
+    tax_group_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      nullable: true
+    is_billable:
+      type: boolean
+      nullable: true
+      example: true
+      description: This fields marks whether a given expense is billable a not.
+    is_reimbursable:
+      type: boolean
+      nullable: true
+      description: This fields marks whether a given expense is reimbursable a not.
+      example: true
+    distance:
+      type: number
+      nullable: true
+      description: Actual distance travelled by employee in travel related expenses.
+    distance_unit:
+      type: string
+      nullable: true
+      description: Unit of distance travelled.
+      enum:
+        - KM
+        - MILES
+        - null
+      example: MILES
+    started_at:
+      $ref: './fields.yaml#/timestamptz_pst'
+    ended_at:
+      $ref: './fields.yaml#/timestamptz_pst'
+    travel_classes:
+      description: |
+        This field is applicable only for `Flight`, `Train`, `Bus`, `Taxi` category.
+        index 0 holds onward journey class, index 1 holds return journey travel class.
```

Comment:
Start with capital I

Reply:
either make `journey class` at both the places or make `journey travel class` at both the places